core/wiki/config/ServerExternalFiltersTags.tid

```diff
@@ -0,0 +1,2 @@
+title: $:/config/Server/ExternalFilters/[tags[]!is[system]sort[title]]
```

Comment:
I don't think that the core should be ship with too many external filters enabled, especially ones that aren't used by the core.

Reply:
That makes sense. Would there be a good spot in the documentation were I could include a section on how people can use the "new" extended server API? Otherwise, I'd expect it to stay a hidden feature.

Reply:
> Would there be a good spot in the documentation were I could include a section on how people can use the "new" extended server API? 

Yes we should update the existing "WebServer API: Get All Tiddlers" docs tiddler.